### PR TITLE
feat: allow storageclass value configure

### PR DESCRIPTION
**Pull request by dongjiang1989**
Add allow storage class value configure

### CODE REVIEW

File: helm/values-acka.yaml [hunk 1]
```diff
@@ -39,14 +39,59 @@ controller:
   update_nls: "false"
   framework_scheduler_names: ahe-scheduler
 storageclass:
+  # lvm StorageClass
   lvm:
-    name: yoda-lvm-default
```
Comment (lihezhong93) [inline helm/values-acka.yaml:43]: do not change storageClass

**Comment by peter-wangxu** [comment]
LGTM

**Comment by lihezhong93** [comment]
LGTM

**Pull request merged**